server/test/controllers/admin/AdminApplicationControllerTest.java

```diff
@@ -126,4 +169,60 @@ public void updateNote_notAdmin() throws Exception {
     Result result = controller.updateNote(request, program.id, application.id);
     assertThat(result.status()).isEqualTo(UNAUTHORIZED);
   }
+
+  // Returns a controller with a faked ProfileUtils to bypass acl checks.
+  AdminApplicationController makeNoOpProfileController() {
+    return new AdminApplicationController(
+        instanceOf(ProgramService.class),
+        instanceOf(ApplicantService.class),
+        instanceOf(ExporterService.class),
+        instanceOf(JsonExporter.class),
+        instanceOf(PdfExporter.class),
+        instanceOf(ProgramApplicationListView.class),
+        instanceOf(ProgramApplicationView.class),
+        instanceOf(ProgramAdminApplicationService.class),
+        instanceOf(ProfileUtilsNoOpTester.class),
+        instanceOf(MessagesApi.class),
+        instanceOf(DateConverter.class),
+        Providers.of(LocalDateTime.now(ZoneId.systemDefault())),
+        instanceOf(FeatureFlags.class));
+  }
+
+  // A test version of ProfileUtils that disable functionality that is hard
+  // to otherwise test around.
+  static class ProfileUtilsNoOpTester extends ProfileUtils {
```

Comment:
I believe we have need for this outside of the class as well. Mind adding it to some test support package?

Reply:
per offline: Will extract it later after it's used more here.  Need to assess the broader usage too, I think only a File class uses ProfileUtils.